AppCenter/AppCenter/Internals/Util/MSPerformSelectorUtil.h

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
+
+@interface MSPerformSelectorUtil : NSObject
```

Comment:
Should probably rename this class something like `MSDispatcherUtil`

Reply:
Renamed